packages/react-theme/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './themes/index';
 export * from './utils/index';
+export * from './global/index';
```

Comment:
```suggestion
export * as globals from './global/index';
```

This should be probably exported with `as` to be accessible as:

```ts
import { globals } from "@fluentui/react-theme";
import { globals } from "@fluentui/react-components";
```

We have already similar PR, #21667. @microsoft/teams-prg let's discuss this in offline as it's not enough export it, it should be also documented.

Reply:
Thanks @layershifter, I've pushed a commit with your suggestion (I just realized I could've just hit the button in the PR).

@miroslavstastny - I was just made aware of your PR #21667 and I think we're on the same track. Tracking some conversation in that PR.